plugin/terminaldocument.vim

```diff
@@ -6,13 +6,18 @@ autocmd BufEnter,BufFilePost * call s:update_document(0)
 " these events might change the nature of the file,
 " so we try to refresh its icon
 autocmd BufWritePost,FileChangedShellPost,ShellCmdPost * call s:update_document(1)
-autocmd VimLeave * call s:set_osc6('')
+
+if &term != 'xterm-ghostty'
+  " For ghostty setting this to empty results in the path being set to "/"
+  " At least before https://github.com/ghostty-org/ghostty/pull/3399
```

Comment:
I see your pull request on Ghostty was merged. I don't know what their release cadence will be like, but perhaps we could just wait a few more days and avoid this condition altogether?

Reply:
Yes, I've dropped this.